### PR TITLE
Added options for enabling SSL

### PR DESCRIPTION
**Pull request by mdolezelova**
Cherry-pick into:

* [x] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

File: guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc
Comment (ekohl) [inline guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc:34]: This should be the default behavior when a root cert is specified.

File: guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc
Comment (ekohl) [inline guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc:37]: This is the default value

**Review by ekohl — APPROVED** [review]
I'm not sure about specifying the default values again. While they don't hurt, as an installer maintainer I'd feel safe leaving them out.

I also checked and there is no option to specify the CA for Candlepin. I don't know how Candlepin handles the CA it trusts.

So this PR is correct.

File: guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc
Comment (mdolezelova) [reply guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc:37]: Hi @ekohl  , thank you for your review! Apologies, I'm not sure whether your comments just state the facts or are meant as requirements for further changes in the file I've edited. Please, are there any additional changes needed so that this PR can be merged? Just FYI, exactly the same changes have been confirmed as correct and already merged to branch 2.5 (Satellite 6.10).  Thank you!

Comment (ekohl) [reply guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc:37]: They go together with:
> I'm not sure about specifying the default values again. While they don't hurt, as an installer maintainer I'd feel safe leaving them out.

This is marking which values I'm talking about.

**Comment by Lennonka** [comment]
Can I merge this?

**Comment by mdolezelova** [comment]
> Can I merge this?

@Lennonka , it would be great. Thank you! just for context, there were two PRs for the same  - one as a bug fix for version 6.10 (2.5) and this one to ensure that future versions are fixed as well. The one for the branch 2.5 was merged, but this one has been pending the approval  for quite some time. I guess it went out of everyone's radar.  So would be great if you can merge it now. Thank you, Lena!

**Comment by Lennonka** [comment]
Cherry-picked:
- 873cf763..ef33f6af  3.1 -> 3.1

**Comment by maximiliankolb** [comment]
> @maximiliankolb Gonna be merging like crazy now, Max! :D

You can start with https://github.com/theforeman/foreman-documentation/pull/887. Just kidding, but comments/approval/concerns are appreciated. 👀

**Comment by Lennonka** [comment]
@maximiliankolb I'm aware of it, but I also have stuff lining up in our Sat backlog. It may take a while.

**Pull request merged**
